examples/js/controls/OrbitControls.js

```diff
@@ -1125,11 +1125,11 @@ THREE.OrbitControls = function ( object, domElement ) {
 	scope.domElement.addEventListener( 'contextmenu', onContextMenu, false );
 
 	scope.domElement.addEventListener( 'mousedown', onMouseDown, false );
-	scope.domElement.addEventListener( 'wheel', onMouseWheel, false );
+	scope.domElement.addEventListener( 'wheel', onMouseWheel, { capture: false, passive: false } );
```

Comment:
I can't find on the internets if `capture` is `false` by default...

Reply:
For context, `useCapture` (the third parameter) is supposed to be `false` by default, but Firefox used to error a decade ago if it was not specified.

Reply:
See [this spec](https://dom.spec.whatwg.org), but the relevant section is non-normative.

Reply:
Thanks!

> capture (a boolean, initially false)

I guess I can remove it from the `options` object then?

Reply:
Since _non-normative_ means the specifications in that section are recommendations, not requirements, I think it would be best if we specify the value we want.

Reply:
I was reading a different section than you were.

>When set to true, options’s capture prevents callback from being invoked when the event’s eventPhase attribute value is BUBBLING_PHASE. When __false (or not present)__, ...

Reply:
So I guess it should just be this?

```js
scope.domElement.addEventListener( 'wheel', onMouseWheel, { passive: false } );
```

PS: I realised I didn't explain the `capture` issue property. Firefox used to crash because the third parameter was not passed.